docfx_yaml/extension.py

```diff
@@ -1351,6 +1350,20 @@ def highlight_md_codeblocks(mdfile):
         mdfile_iterator.write(new_content)
 
 
+def prepend_markdown_header(filename, mdfile_iterator):
```

Comment:
I thought I commented with this before, but I don't see it. Consider using type hints instead of naming the arg `..._iterator`.